sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -325,6 +325,25 @@ def test_binomial():
 
     assert binomial(a, b).is_nonnegative is True
 
+    # issue #13980 and #13981
+    assert binomial(-7, -5) == 0
+    assert binomial(-23, -12) == 0
+    assert binomial(13/2, -10) == 0
+
+    assert binomial(-49, -51) == binomial(-49, -51)
+    assert binomial(19, -7/2).rewrite(gamma) == -68719476736/(911337863661225*pi.evalf())
+    assert binomial(-3, -7/2).rewrite(gamma) == zoo
```

Comment:
Why is the rewrite necessary? `binomial(-integer, -noninteger)` should give zoo.